src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -71,8 +72,7 @@
     @NotNull
     private org.opentestsystem.ap.common.model.Item itemJson;
 
-    @OneToMany
-    @JoinColumn(name = "item_key")
+    @OneToMany(mappedBy = "standardIdKey.item", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
So this is the magic that took me way too long to figure out. The standardIdKey.item is enough for JPA/Hibernate to know that you mean look at the StandardIdKey's item property within StandardId. Most of the examples around using OneToMany don't use composite keys.

Long term we may want to consider removing the BaseItem from the StandardIdKey and instead save standard ids with the item by using this field.  The reason being is we have a bidirectional and one could himself or herself in a circular loop.  And we don't fetch standard ids without their item.

Reply:
I agree that would be better - it is a pretty easy change on the ingest side.